tests/bindings/python/test_importer.py

```diff
@@ -0,0 +1,40 @@
+#
+# Tests the Importer class bindings
+#
+import unittest
+
+
+class ValidatorTestCase(unittest.TestCase):
+
+    def test_create_destroy(self):
+        from libcellml import Importer
+
+        x = Importer()
+        del(x)
```

Comment:
This should actually read `del x`. This what PyCharm recommends and what we should, in fact, do everywhere in our Python code.

Reply:
A quick look through the python tests shows that we have 59 instances of `del(something)` ... and 4 of `del something`... perhaps another item for the housekeeping issue?

Reply:
Fair enough, just added to issue #402 (as well as fix this particular occurrence since it's new code :)).

Reply:
7b7ad654